app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailFragment.java

```diff
@@ -145,7 +130,6 @@ public void onRefreshComplete() {
 
     @Override
     public void showEvent(Event event) {
-        listener.onEventLoaded(event, false);
```

Comment:
Yeah, I wanted to discuss this. Shouldn't main presenter load the event independently from detail presenter?